src/app/collective-rewards/rewards/utils/formatMetrics.ts

```diff
@@ -1,16 +1,11 @@
-import { formatCurrency, toFixed } from '@/lib/utils'
-import { formatUnits } from 'viem'
+import { formatCurrency } from '@/app/collective-rewards/utils/formatting/formatter'
 
-export const formatMetrics = (amount: number, price: number, symbol: string, currency: string) => ({
-  amount: `${toFixed(amount)} ${symbol}`,
-  fiatAmount: `= ${currency} ${formatCurrency(amount * price, currency)}`,
+export const formatMetrics = (
+  amount: number | bigint,
+  price: number | bigint,
+  symbol: string,
+  currency: string,
+) => ({
+  amount: `${formatCurrency(amount, symbol) || 0} ${symbol}`,
+  fiatAmount: `= ${currency} ${formatCurrency(Number(formatCurrency(amount, symbol, { thousandsSeparator: '' })) * Number(price) || 0, currency)}`,
```

Comment:
hey @fran thanks for finishing this. the `thousandsSeparator` override is not needed here, as fiat [defaults](https://github.com/RootstockCollective/dao-frontend/pull/446/files#diff-f662adaedcc0342f033320fb578c95c7c05b17674d2f2da3ab2799ad529ebe44R18) to the same empty string.

Reply:
For RBTC is ',', do when I send the symbol it returns a 0,02 number that returns Nan

Reply:
And if we send the currency instead of the symbol, we will be applying the format of USD instead of rbtc or rif, which causes issues with the approximation, since we are rounding those with different decimals